addons/mail/models/mail_mail.py

```diff
@@ -20,6 +21,10 @@
 
 _logger = logging.getLogger(__name__)
 
+BASE_64_SIZE_OVERHEAD_RATIO = 8 / 6
+EMAIL_SIZE_SECURITY_MARGIN_BYTES = 50 * 1024
```

Comment:
shouldn't be this in  percent of the limit ? (e.g. for extreme cases, if you server supports only 30ko, it's strange to use a security margin of 50ko)

Reply:
This is for the estimation of the email size to take into account the format that glue all part together (headers, body, attachment). I have reduced it to 10Ko. I think it is small enough regarding the size commonly authorized: almost always 10Mo. The only thing it will cause is to set the attachment as link in the email while it could have sent it as regular attachment because of an over-estimation of 10Ko.